resources.go

```diff
@@ -83,16 +88,22 @@ const (
 	azureRecoveryServices    = "recoveryservices"    // Recovery Services
 	azureRedis               = "redis"               // RedisCache
 	azureRelay               = "relay"               // Relay
-	azureRole                = "role"                // Azure Role
 	azureScheduler           = "scheduler"           // Scheduler
 	azureSecurityCenter      = "securitycenter"      // Security Center
+	azureServiceBus          = "servicebus"          // ServiceBus
 	azureServiceFabric       = "servicefabric"       // Service Fabric
 	azureSearch              = "search"              // Search
 	azureSignalr             = "signalr"             // SignalR
 	azureSQL                 = "sql"                 // SQL
 	azureStorage             = "storage"             // Storage
 	azureStreamAnalytics     = "streamanalytics"     // StreamAnalytics
-	azureTrafficManager      = "trafficmanager"      // Traffic Manager
+
+	// Legacy Module Names
+	azureLegacyRole             = "role"               // Azure Role
+	azureLegacyMSI              = "msi"                // Managed Service Identity (MSI)
+	azureLegacyManagementGroups = "managementgroups"   // Management Groups (Legacy)
+	azureLegacyMgmtResource     = "managementresource" // Management Resource (Legacy)
+	azureLegacyTrafficManager   = "trafficmanager"     // Traffic Manager
```

Comment:
can you be consistent here with the `(legacy)` moniker?  Having some have it and some not actually confuses me in terms of wondering if they're different somehow.

Reply:
done!